Sources/VLCSettingsController.m

```diff
@@ -125,12 +150,46 @@ - (void)updateUIAndCoreSpotlightForPasscodeSetting:(BOOL)passcodeOn
 
 #pragma mark - IASKSettings delegate
 
-- (void)settingsViewController:(IASKAppSettingsViewController*)sender buttonTappedForSpecifier:(IASKSpecifier*)specifier
+- (IBAction)pushAboutViewController:(id)sender
 {
     VLCAboutViewController *aboutVC = [[VLCAboutViewController alloc] init];
     [self.navigationController pushViewController:aboutVC animated:YES];
 }
 
+- (CGFloat) settingsViewController:(id<IASKViewController>)settingsViewController tableView:(UITableView *)tableView heightForHeaderForSection:(NSInteger)section {
+    return section == 0 ? 0 : SETTINGS_HEADER_HEIGHT;
+}
+
+- (UIView *)settingsViewController:(id<IASKViewController>)settingsViewController tableView:(UITableView *)tableView viewForHeaderForSection:(NSInteger)section
+{
+    UIView *header = [[UIView alloc] initWithFrame:CGRectMake(HEADER_OFFSET, 0, tableView.bounds.size.width, SETTINGS_HEADER_HEIGHT)];
+    UILabel *headerLabel = [[UILabel alloc] init];
+    
+    headerLabel.font = PresentationTheme.current.fonts.sectionHeader;
+    headerLabel.text = [settingsViewController.settingsReader titleForSection:section];
+    headerLabel.textColor = PresentationTheme.current.colors.cellTextColor;
+    headerLabel.translatesAutoresizingMaskIntoConstraints = false;
+    [header addSubview:headerLabel];
+    
+    UILayoutGuide *margins = [header layoutMarginsGuide];
+    [NSLayoutConstraint activateConstraints:@[
+                                              [headerLabel.leadingAnchor constraintEqualToAnchor:margins.leadingAnchor constant:8],
+                                              [headerLabel.trailingAnchor constraintEqualToAnchor:margins.trailingAnchor],
+                                              [headerLabel.topAnchor constraintEqualToAnchor:margins.topAnchor],
+                                              [headerLabel.bottomAnchor constraintEqualToAnchor:margins.bottomAnchor]
+                                              ]];
+    
+    UIView *separator = [[UIView alloc] initWithFrame:CGRectMake(HEADER_OFFSET, 0, header.frame.size.width - (HEADER_OFFSET * 2), 2)];
+    separator.backgroundColor = PresentationTheme.current.colors.settingsSeparatorColor;
+    [header addSubview:separator];
```

Comment:
This will "break" and by that I mean the separator will be too short when you rotate the view since you set a static frame